fahrschein/src/main/java/org/zalando/fahrschein/NakadiClientBuilder.java

```diff
@@ -48,6 +67,10 @@ public NakadiClientBuilder withCursorManager(CursorManager cursorManager) {
         return new NakadiClientBuilder(baseUri, objectMapper, authorizationProvider, clientHttpRequestFactory, cursorManager);
     }
 
+    public NakadiClientBuilder withTracingInterceptor(List<RequestHandler> requestHandlers) {
```

Comment:
```suggestion
    public NakadiClientBuilder withRequestHandlers(List<RequestHandler> requestHandlers) {
```

 but I would actually prefer `addRequestHandler(RequestHandler requestHandler)` and iteratively regstering them.

Reply:
> but I would actually prefer addRequestHandler(RequestHandler requestHandler) and iteratively regstering them.

Why not both? Having a fluent builder is nice for times where the user thinks they have all the handlers at construction time. I'd much rather construct with `builder.withRequestHandlers(myHandlers).build()` than construct-then-modify using `addRequestHandler` when I already have `myHandlers` defined.